File: codeWars/katas/hamming_distance/hamming_distance.py
# Hamming Distance


def hamming_distance(a, b):
    if len(a) != len(b):
        raise ValueError("Strings must have equal length")

    distance = 0
    for i in range(len(a)):
        if a[i] != b[i]:
            distance += 1

    return distance


a = "I like turtles"
b = "I like turkeys"
print(hamming_distance(a, b))  # Output: 3

File: codeWars/katas/hamming_distance/hamming_distance.js
// Hamming Distance


function hammingDistance(a, b) {
    if (a.length !== b.length) {
        throw new Error("Strings must have equal length");
    }

    let distance = 0;
    for (let i = 0; i < a.length; i++) {
        if (a[i] !== b[i]) {
            distance++;
        }
    }

    return distance;
}

const a = "I like turtles";
const b = "I like turkeys";
console.log(hammingDistance(a, b)); // Output: 3
